src/main/resources/messages.properties

```diff
@@ -62,6 +62,8 @@ exception.advertisement-image.id.not-found=Not valid ID of the advertisement or
 exception.image.not-existed-id=Received not existed images IDs: {0}
 exception.invalid.locations.file.creating.data=Invalid data to create database initialization file for locations
 exception.emailing.registration=It was unexpected problem during sending a registration email. Try again a bit later
+exception.email.not-found=Email not found
```

Comment:
Non-informative exception message. As for me, it's related to the validated code more than email?